### PR TITLE
Support federated registry with InstantOn

### LINKED ISSUE #23706: InstantOn federatedRegistry-1.0

### LINKED ISSUE #23707: InstantOn ldapRegistry-3.0

### PR DESCRIPTION
**Pull request by tjwatson**

### CODE REVIEW

**Comment by tjwatson** [comment]
#build

**Comment by tjwatson** [comment]
#build

**Comment by tjwatson** [comment]
Note to `@OpenLiberty/message-reviewer` this PR does not actually have any new messages.  The original commits included message updates from another PR.

**Comment by tjwatson** [comment]
#build

**Comment by posmith** [comment]
L2 message review not required

**Review by jvanhill — APPROVED** [review]

File: dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/URAPIs_Federation_2LDAPsTest.java [hunk 1]
```diff
@@ -0,0 +1,947 @@
...
+            assertNotNull("Application userRegistry does not appear to have started.",
+                          s.waitForStringInLog("CWWKZ0001I:.*userRegistry"));
+            // Add LDAP variables to server.env file
+            try {
+                // gathering info from bootstrap.properties first
+                LDAPUtils.addLDAPVariables(server);
+                Properties bootstrapProps = server.getBootstrapProperties();
+                Map<String, String> envMap = new HashMap<>();
+                // capture the bootstrap.properties values into the server.env for ldap settings
+                for (Entry<Object, Object> entry : bootstrapProps.entrySet()) {
```
Comment (sebratton) [inline dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/URAPIs_Federation_2LDAPsTest.java:120]: Shouldn't the env with host and port settings be made available after server checkpoint?

Comment (tjwatson) [reply dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/URAPIs_Federation_2LDAPsTest.java:120]: > Shouldn't the env with host and port settings be made available after server checkpoint?

This is within the `beforeRestoreLambda` lambda being passed to the `CheckpointInfo`.  I could extract the lambda out to a local variable to make that more clear.  But this happens after checkpoint.

**Comment by tjwatson** [comment]
#libby

File: dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/URAPIs_Federation_2LDAPsTest.java (hunk 1 above)
Comment (sebratton) [reply dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/URAPIs_Federation_2LDAPsTest.java:120]: My bad. I missed that it was part of the Lambda.

**Review by sebratton — APPROVED** [review]

**Pull request merged**